src/librustc_resolve/macros.rs

```diff
@@ -412,7 +428,7 @@ impl<'a> Resolver<'a> {
                             attrs
                         });
                     }
-                    return Err(Determinacy::Undetermined);
```

Comment:
Hmm, I think the return was actually correct, but misplaced - we *can* return once we've found an inert attribute and processed it, we don't need to go through remaining traits and possibly repeat the process, so the return should go inside `if inert_attrs.contains(&attr_name) { ... }`

Reply:
Oh right yes indeed!

Reply:
should be updated now!